src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,75 @@
+import classNames from 'classnames';
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types/Person';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  people: Person[],
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+
+  return (
+    <>
+      <table
+        data-cy="peopleTable"
+        className="table is-striped is-hoverable is-narrow is-fullwidth"
+      >
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Sex</th>
+            <th>Born</th>
+            <th>Died</th>
+            <th>Mother</th>
+            <th>Father</th>
+          </tr>
+        </thead>
+
+        <tbody>
+          {people.map(person => {
+            const {
+              sex,
+              motherName,
+              fatherName,
+              born,
+              died,
+              mother,
+              father,
+            } = person;
+
+            return (
+              <tr
+                key={person.slug}
+                data-cy="person"
+                className={classNames({
+                  'has-background-warning': person.slug === slug,
+                })}
+              >
+                <td>
+                  <PersonLink person={person} />
+                </td>
+
+                <td>{sex}</td>
+                <td>{born}</td>
+                <td>{died}</td>
+                <td>
+                  {mother
+                    ? <PersonLink person={mother} />
+                    : motherName || '-'}
```

Comment:
it isn't a good idea to use ternary with binary operators this way. think about refactoring. you can use helper variable